documentation/docs/guide/GettingStarted/deploy_locally.md

```diff
@@ -46,11 +61,14 @@ docker compose --file docker-compose-init.yaml run self-signed-certificate-gener
 ![keychainSnippet](../../assets/keychainSnippet.png)
 
 ::: tip
-If this cert is not working, try restarting your MAC, however there might be a less drastic way of making this work.
+If this cert is not working, try refreshing the Keychain or verifying the certificate installation before considering a Mac restart as a last resort.
 :::
 
 ::: warning
-WordPress will now run on localhost port 443, which is exactly the same port that Kubernetes runs on. Therefore local Kubernetes and WordPress can't be running at the same time.
+WordPress will now run on localhost port 443, which is exactly the same port that Kubernetes runs on. Therefore, local Kubernetes and WordPress can't be running at the same time.
+
+::: tip Workaround
+If you need both services running simultaneously, consider changing the port for one of them. For example, you can modify the `docker-compose.yaml` file to use a different port for WordPress by updating the `ports` section under the `nginx` service.
```

Comment:
actually this is not true, you can't do this, and the reason you can't is because NGINX is looking for 443, if you change this without change the nginx config it won't work. (please remove this tip),